src/AdoNetCore.AseClient/AseError.cs

```diff
@@ -95,12 +96,25 @@ public int TranState
         {
             get; internal set;
         }
-        
+
 
         /// <summary>
         /// The error message that comes from the Adaptive Server server.
         /// </summary>
-        public bool IsFromServer 
+        public bool IsFromServer
+        {
+            get; internal set;
+        }
+
+        /// <summary>
+        /// The error message that comes from the Adaptive Server Backup server.
+        /// </summary>
+        public bool IsFromBackupServer
+        {
+            get; internal set;
+        }
+
+        public BackupServerErrorInfo BackupServerErrorInfo
```

Comment:
I don't think this is necessary either, since it also relates to parsing the messages.